### PR TITLE
Comment out home page test elements

### PR DESCRIPTION
**Pull request by mohe2015**

### CODE REVIEW

File: js/src/main/scala/webapp/entity/UIAttribute.scala [hunk 1]
```diff
@@ -309,7 +309,7 @@ class UISelectAttribute[EntityType, AttributeType](
       },
       attr.map(getEditString(_)),
       searchEnabled,
-      span("Nothing found..."),
+      span("Nothing found. Maybe you need to create first?"),
```
Comment (mohe2015) [inline js/src/main/scala/webapp/entity/UIAttribute.scala:312]: This is there because I want to replace it by a link to create a new entity

**Comment by mohe2015** [comment]
Closes https://github.com/reform-org/reform/issues/384

**Comment by Oshgnacknak** [comment]
I would have moved it to a `/debug` route that's not linked anyware

**Comment by mohe2015** [comment]
> I would have moved it to a `/debug` route that's not linked anyware

This was just a quick fix before our user testing. Feel free to improve

**Pull request merged**